release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1948,6 +1948,91 @@ In the following tables, features are marked with the following statuses:
++
+[source,terminal]
+----
+$ cd ~/clusterconfigs/openshift
+$ vim openshift-worker-0.yaml
+----
++
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  annotations:
+    bmac.agent-install.openshift.io/installer-args: '["--append-karg", "ip=<static_ip>::<gateway>:<netmask>:<hostname_1>:<interface>:none", "--save-partindex", "1", "-n"]' <1> <2> <3> <4> <5>
+    inspect.metal3.io: disabled
+    bmac.agent-install.openshift.io/hostname: <fqdn> <6>
+    bmac.agent-install.openshift.io/role: <role> <7>
+
+  generation: 1
+  name: openshift-worker-0
+  namespace: mynamespace
+spec:
+  automatedCleaningMode: disabled
+  bmc:
+    address: idrac-virtualmedia://<bmc_ip>/redfish/v1/Systems/System.Embedded.1 <8>
+    credentialsName: bmc-secret-openshift-worker-0
+    disableCertificateVerification: true
+  bootMACAddress: 94:6D:AE:AB:EE:E8
+  bootMode: "UEFI"
+  rootDeviceHints:
+    deviceName: /dev/sda
+----
+For the `ip` settings, replace:
+<1> `<static_ip>` with the static IP address for the node, for example, `192.168.1.100`
+<2> `<gateway>` with the IP address of your network's gateway, for example, `192.168.1.1`
+<3> `<netmask>` with the network mask, for example, `255.255.255.0`
+<4> `<hostname_1>` with the node's hostname, for example, `node1.example.com`
+<5> `<interface>` with the name of the network interface, for example, `eth0`
+<6> `<fqdn>` with the fully qualified domain name of the node
+<7> `<role>` with `worker` or `master` to reflect the node's role
+<8> `<bmc_ip>` with with the BMC IP address and the protocol and path of the BMC, as needed.
+
+.. Save the file to the `clusterconfigs/openshift` directory.
+
+.. Create the cluster.
+
+. When installing with the {ai-full}, before installation modify each node's installer arguments using the API to append the network settings as `ip` installer arguments. For example:
++
+[source,bash]
+----
+$ curl https://api.openshift.com/api/assisted-install/v2/infra-envs/${infra_env_id}/hosts/${host_id}/installer-args \
+-X PATCH \
+-H "Authorization: Bearer ${API_TOKEN}" \
+-H "Content-Type: application/json" \
+-d '
+    {
+      "args": [
+        "--append-karg",
+        "ip=<static_ip>::<gateway>:<netmask>:<hostname_1>:<interface>:none", <1> <2> <3> <4> <5>
+        "--save-partindex",
+        "1",
+        "-n"
+      ]
+    }
+  ' | jq
+----
+For the previous network settings, replace:
+<1> `<static_ip>` with the static IP address for the node, for example, `192.168.1.100`
+<2> `<gateway>` with the IP address of your network's gateway, for example, `192.168.1.1`
+<3> `<netmask>` with the network mask, for example, `255.255.255.0`
+<4> `<hostname_1>` with the node's hostname, for example, `node1.example.com`
+<5> `<interface>` with the name of the network interface, for example, `eth0`. 
+
+Contact Red Hat Support for additional details and assistance.
+
+(link:https://issues.redhat.com/browse/OCPBUGS-17895[*OCPBUGS-17895*])
```

Comment:
This bug has a RH only level of security. Is there a public version of this bug? Looks like that is why your PR is failing too. Taking off merge review for now because we typically do not put bugs in that are behind a login in JIRA.

Reply:
Fixed.